jobs/JGEFS_WAVE_STAT_PNT

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/jjob_header.sh" -e "wave_stat" -c "base wave wave_stat wave_stat_pnt"
+source "${USHgfs}/wave_domain_grid.sh"
+
+# Set COM Paths
+
+for grid in ${wavepostGRD}; do
+  process_grdID "${grid}"
+  prod_dir="COMOUT_WAVE_STATION_ENS"
+  MEMDIR="ensstat"  YMD=${PDY} HH=${cyc} declare_from_tmpl -rx "${prod_dir}:COM_WAVE_STATION_TMPL"
+  if [[ ! -d "${!prod_dir}" ]]; then mkdir -m 775 -p "${!prod_dir}"; fi
+done
```

Comment:
Where is `grid` used in this COM variable declaration? The [`COM_WAVE_STATION_TMPL` doesn't have a grid setting](https://github.com/NOAA-EMC/global-workflow/blob/develop/dev/parm/config/gfs/config.com#L83) in it. Also, what is `process_grdID "${grid}"` doing?
I've removed the loop here since it doesn't appear needed but please keep it if so, I may just not understand the intention. :)
```suggestion
MEMDIR="ensstat" YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
  COMOUT_WAVE_STATION_ENS:COM_WAVE_STATION_TMPL
if [[ ! -d "${COMOUT_WAVE_STATION_ENS}" ]]; then
  mkdir -m 775 -p "${COMOUT_WAVE_STATION_ENS}"
fi
```